libhugetlbfs: TESTS_64 is not defined for ia64, check when trying to install it

That was for libhugetlbfs version 2.12 (TEST_64 is not defined for ia64),
same seem to be the case for x86_64 in version 2.15.
-> Revert the installation of this test totally.

Signed-off-by: Thomas Renninger <trenn@suse.de>

---
 tests/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: libhugetlbfs-2.21/tests/Makefile
===================================================================
--- libhugetlbfs-2.21.orig/tests/Makefile
+++ libhugetlbfs-2.21/tests/Makefile
@@ -295,7 +295,7 @@ obj64/install:
 	$(INSTALL) -m 755 wrapper-utils.sh $(DESTDIR)$(INST_TESTSDIR64)/obj64
 	$(INSTALL) -m 755 $(HELPERS:%=obj64/%) $(DESTDIR)$(INST_TESTSDIR64)/obj64
 	$(INSTALL) -m 755 $(HELPER_LIBS:%=obj64/%) $(DESTDIR)$(INST_TESTSDIR64)/obj64
-	$(INSTALL) -m 755 $(TESTS_64:%=obj64/%) $(DESTDIR)$(INST_TESTSDIR64)/obj64
+
 	$(INSTALL) -m 755 run_tests.py $(DESTDIR)$(INST_TESTSDIR64)
 
 install: $(OBJDIRS:%=%/install)
